test/terra/backends/aer_simulator/test_options.py

```diff
@@ -46,7 +46,7 @@ def test_seed_simulator_option_measure(self, method, device):
         qc.h([0, 1])
         qc.reset(0)
         qc.measure_all()
-        qc = transpile(qc, backend)
+        qc = transpile(qc, backend, optimization_level=0)
```

Comment:
Why do we need to use level 0 here?

Reply:
with transpile>0, H gate is converted to P gate in extended_stabilizer method that takes very long time to run this circuit. In backendV1, H gate was not converted

Reply:
I remember this now, the issue tracking this in qiskit is: https://github.com/Qiskit/qiskit/issues/10568